File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/exponential/Exp1OverZ.java
package net.fwitz.math.main.binary.complex.functions.exponential;

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class Exp1OverZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("exp(1/z)")
                .computeFn(z -> z.inverse().exp())
                .domainX(-1, 1)
                .domainY(-1, 1)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/exponential/Exp1OverZ.kt
package net.fwitz.math.main.binary.complex.functions.exponential

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object Exp1OverZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("exp(1/z)")
        .computeFn { z -> z.inverse.exp }
        .domainX(-1.0, 1.0)
        .domainY(-1.0, 1.0)
        .render()
}